### DISCUSSION TITLE
M4B converted files have ridiculously long durations in iTunes

### DISCUSSION THREAD

**Discussion by PiousVenom** [openaudible/openaudible#1386] [Q&A]
I've got about 20 ish books from Audible that I've converted to M4B and added to iTunes, and their durations are 360+ hours. I'm certain this isn't an OpenAudible problem. I was just curious if anyone else has experienced this? Was a solution found? It plays and then just has SO much silence. LOL.

**Comment by oboe45** [comment]
I have more than 1000 books from Audible, cinverted to M4b using Open
Audible, all the books converted correctly and all are listenable from end
to end - no issues whatsoever.

**Reply by PiousVenom** [reply]
Like I said, I'm more than certain it isn't an OpenAudible issue. If I did, I'd have created a bug instead. I can see in the file that the duration is correct. It's just upon adding to iTunes it has the increased duration. Just asking if anyone else has experienced this and if they'd found a solution.

**Comment by oboe45** [comment]
I used to add them to Itunes and did not notice anything odd.  Since Itunes
has become less and les useful over the past years, I put all my M4b (and
M4a) files onto the Audiobookshelf app and access (play) from there. If you
haven't had a look at Audiobookshelf - check it out, its free and well
maintained and *very* rich in features.

**Reply by PiousVenom** [reply]
Do you mean this app?

https://www.audiobookshelf.org/

Seems they are full on the beta 😟

**Comment by oboe45** [comment]
Yes, that's it! I use the beta version on Ipad and I phone, but I also use
the full server version - running on a synology and/or Qnap NAS as a docker
app.  The full version is the best and has a lot more features than the IOS
version, but all versions are outstanding.  Used in conjunction with the
Tailscale app (another excellent freebee), I have the ability to access my
entire M4b library remotely, securely on any of my devices.  The deficiency
is the lack of an Alexa interface, so I cannot play books by spoken command
as I can on the Audible app.  Hopefully I can figure out a Siri way of
doing it eventually.  The docker install is well supported on a couple of
"how to" sites and seems bulletproof - I have been using
Audible/Audiobookhelf and open audible for a very long time - all great
products
